Gems/AWSCore/cdk/README.md

```diff
@@ -65,6 +65,17 @@ them to your `setup.py` file and rerun the `pip install -r requirements.txt`
 command.
 
 ## Optional Features
+
+### Automatic S3 and DynamoDB Cleanup
+The S3 bucket and Dynamodb created by the sample will be left behind as the CDK defaults to retaining such storage (both default to RemovalPolicy.RETAIN). To delete
+the storage resources created when using CDK destroy, use the following commands to synthesize and destroy the CDK application.
+```
+cdk synth -c remove_all_storage=true --all
+cdk deploy -c remove_all_storage=true --all
```

Comment:
The context variable seems a little weird since it says to remove all storage even for deployment. Maybe rename it to something like remove_all_storage_on_deletion?

Reply:
Agreed it looks a little odd on the synth/deployment line. Will update to a better context variable name.